delegates.txt

```diff
@@ -41,6 +41,7 @@ Augustus Yuan (AYN)
 Avik Chaudhuri (AVC)
 Axel Rauschmayer (ARR)
 Aysegul Yonet (AYS)
+Ben Allen (BEN)
```

Comment:
our typical convention would have this be `BAN`, is that ok?

Reply:
Shoot, I knew snagging BEN was too good to be true. BAN is fine, though if possible BJA would be preferable

Reply:
Also, where's the matching reflector issue for this addition? Can you have your Ecma member company point of contact file one?

Reply:
@ljharb apologies, was waiting on the PR link before posting.

Reply:
Nudging this forwards...

```suggestion
Ben Allen (BAN)
```